$NetBSD$

Don't mess with CPPFLAGS and LDFLAGS as pkgsrc handles them

--- configure.orig	2012-05-06 23:37:04.000000000 +0200
+++ configure	2012-05-06 23:37:14.000000000 +0200
@@ -3145,8 +3145,8 @@
 	# XXX: not sure if /usr/local is necessary.
 	# XXX: linux libwrap needs -lnsl. configure should check for
 	#      existence of libnsl instead of hard-coding
-	CPPFLAGS="$CFLAGS -I/usr/local/include"; export CPPFLAGS
-	LDFLAGS="$LDFLAGS -L/usr/local/lib -L/lib"; export LDFLAGS
+	# CPPFLAGS="$CFLAGS -I/usr/local/include"; export CPPFLAGS
+	# LDFLAGS="$LDFLAGS -L/usr/local/lib -L/lib"; export LDFLAGS
 	LIBS="-lnsl -lcrypt $LIBS"; export LIBS
         $as_echo "#define LINUX 1" >>confdefs.h
 
@@ -3154,15 +3154,15 @@
 	# XXX: does linux need glibc: -DGLIBC
     ;;
     *mips* )
-	CPPFLAGS="$CFLAGS -I/usr/local/include"; export CPPFLAGS
-	LDFLAGS="$LDFLAGS -L/usr/local/lib"; export LDFLAGS
+	# CPPFLAGS="$CFLAGS -I/usr/local/include"; export CPPFLAGS
+	# LDFLAGS="$LDFLAGS -L/usr/local/lib"; export LDFLAGS
 	LIBS="-lcrypt $LIBS"; export LIBS
         $as_echo "#define MIPS 1" >>confdefs.h
 
     ;;
     * )
-	CPPFLAGS="$CFLAGS -I/usr/local/include"; export CPPFLAGS
-	LDFLAGS="$LDFLAGS -L/usr/local/lib"; export LDFLAGS
+	# CPPFLAGS="$CFLAGS -I/usr/local/include"; export CPPFLAGS
+	# LDFLAGS="$LDFLAGS -L/usr/local/lib"; export LDFLAGS
     ;;
 esac
 
